gstchecksumsink.c

```diff
@@ -435,7 +449,7 @@ gst_cksum_image_sink_render (GstBaseSink * sink, GstBuffer * buffer)
     n_planes = GST_VIDEO_FRAME_N_PLANES (&frame);
 
     for (plane = 0; plane < n_planes; plane++) {
-      gpointer pd = GST_VIDEO_FRAME_PLANE_DATA (&frame, plane);
+      guint8 *pd = GST_VIDEO_FRAME_PLANE_DATA (&frame, plane);
```

Comment:
frame->data is stated as type gpointer, maybe we don't need to change it? Is it better to use a forced cast if there is a reason to pass frame->data to a guint8 pointer?

Reply:
fixed, thanks.